Include/pymath.h

```diff
@@ -27,6 +27,10 @@
 #define Py_MATH_TAU 6.2831853071795864769252867665590057683943L
 #endif
 
+/* Py_IS_NAN, Py_IS_INFINITY and Py_IS_FINITE are deprecated
+ * since CPython 3.14.
+ */
+
 // Py_IS_NAN(X)
 // Return 1 if float or double arg is a NaN, else 0.
```

Comment:
Maybe repeat the soft deprecation in the comment of each macro, to ease maintenance (if code is moved for example).

Reply:
Usually we put the deprecation on each function with Py_DEPRECATED(). Add a sentence in comment like:

"Macro soft deprecated since Python 3.14."

Reply:
Hmm, hardly these macros could be moved alone.

Reply:
These comments are mostly for CPython maintainers, isn't?  Important info is: *when* and *how* was deprecated this API.  (Py_DEPRECATED answer to the first question only.)  Workarounds should be obvious for readers, but in a hard case we have a news entry that document that.  So, my version: "Soft deprecated since Python 3.14."

Reply:
No, comments are for users of the API.